### PR TITLE
CSV translation import: update index, refs #8572

### PR DESCRIPTION
**Pull request by mcantelon**
Added search index updating during CSV import of information object
translations.

### CODE REVIEW

File: lib/task/import/csvImportTask.class.php [hunk 1]
```diff
@@ -1056,6 +1056,13 @@ public function execute($arguments = array(), $options = array())
         {
           $this->handleDigitalObjectPath($self, $path);
         }
+
+        // If importing a translation, re-save entire information object so translation will be indexed
+        if ($self->object instanceof QubitInformationObjectI18n && !$search->searchIndexingDisabled)
```
Comment (sbreker) [inline lib/task/import/csvImportTask.class.php]: Does this respect the --index flag? I can't see where $search is getting set.  

The index flag is assigned to the $import object here:
https://github.com/artefactual/atom/blob/qa/2.6.x/lib/task/import/csvImportTask.class.php#L1062

Comment (mcantelon) [reply lib/task/import/csvImportTask.class.php]: Oops... nice catch! Fixed!

**Comment by mcantelon** [comment]
Updated if you want to take another look... thanks @sbreker !

**Review by sbreker — APPROVED** [review]
Looks good @mcantelon!  Thanks for making that change!

**Pull request merged**